src/rendering/renderers/shared/startup/HelloSystem.ts

```diff
@@ -24,15 +25,12 @@ export class HelloSystem implements System<HelloSystemOptions>
             ExtensionType.CanvasSystem,
         ],
         name: 'hello',
-        priority: 0,
+        priority: -2,
```

Comment:
Priority change intentional?

Reply:
yep! this requires the context to exist to get the correct WebGL version.